esrally/rally.py

```diff
@@ -46,7 +46,7 @@ def positive_number(v):
 
     parser = argparse.ArgumentParser(prog=PROGRAM_NAME,
                                      description=BANNER + "\n\n You Know, for Benchmarking Elasticsearch.",
-                                     epilog="Find out more about Rally at %s" % console.format.link(DOC_LINK),
+                                     epilog="Find out more about Rally at %s" % console.format.link(doc_link()),
```

Comment:
Given the opportunity you could replace `%s` and `%` here with `format()` for standardization.

Reply:
Good point. Will do.